pyproject.toml

```diff
@@ -10,10 +10,22 @@ enable_flake8 = true
 enable_isort = true
 enable_mypy = true
 mypy_preset = "very_strict"
-py_version = "py37"
+py_version = "py38"
 isort_known_first_party = ["fakes", "pysen"]
 [[tool.pysen.lint.mypy_targets]]
-  paths = [".", "tests/"]
+  paths = [
+    "pysen/",
+    "setup.py",
+    "tests/",
+  ]
+[[tool.pysen.lint.mypy_targets]]
+  paths = ["examples/advanced_example"]
+[[tool.pysen.lint.mypy_targets]]
+  paths = ["examples/simple_package"]
+[[tool.pysen.lint.mypy_targets]]
+  paths = ["examples/plugin_example"]
+[[tool.pysen.lint.mypy_targets]]
+  paths = ["examples/sync_cmdclass_pyproject"]
```

Comment:
As mypy 0.800+ changes the way to find target sources, I updated this section as described in our FAQ https://github.com/pfnet/pysen#frequently-asked-questions .